hyperspy/drawing/signal.py

```diff
@@ -23,6 +23,8 @@
 from mpl_toolkits.axes_grid1 import make_axes_locatable
 from traits.api import Undefined
 
+from hyperspy.drawing.utils import set_axes_decor
+
 
 def _plot_quiver_scatter_overlay(image, axes_manager,
```

Comment:
Actually, it'll be a good idea to remove this function rather than modifying it as it is not used anywhere.